src/main/java/mate/academy/springboot/datajpa/dto/CategoryRequestDto.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.springboot.datajpa.dto;
+
+import lombok.Data;
+
+@Data
+public class CategoryRequestDto {
+    private String name;
+
+    public String getName() {
+        return name;
+    }
```

Comment:
You already use lombok data annotation, it generates all setters and getters for you.